### PR TITLE
Add Python3.9 support.

### PR DESCRIPTION
**Pull request by hramezani**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -3,10 +3,7 @@ envlist =
     docs
     style
     readme
-    py{36,37,38}-dj22-sqlite
-    py{36,37,38}-dj{30,31}-sqlite
-    py{36,37,38}-djmaster-sqlite
-    py{37,38}-dj{22,30,31}-{postgresql,mysql}
+    py{36,37,38,39}-dj{22,30,31,master}-{sqlite,postgresql,mysql}
```
Comment (matthiask) [inline tox.ini:6]: I'd like to ask you to revert this change. Upcoming Django versions may drop support for Python versions and while the previous format is a bit more verbose it's a bit more obvious to me how to manage/remove individual combinations.

**Review by matthiask — CHANGES_REQUESTED** [review]
Looks good to me (apart from the comment above)

File: tox.ini (hunk 1 above)
Comment (hramezani) [reply tox.ini]: I reverted this change and add `39` to the end of each list

**Comment by hramezani** [comment]
> Looks good to me (apart from the comment ~above~below)

I address your comment and also add Python and Django supported version badges to `README.rst` in a separate commit.

**Comment by matthiask** [comment]
Thanks!
